scripts/upgrade

```diff
@@ -21,31 +21,35 @@ path_url=$(ynh_app_setting_get --app=$app --key=path)
 admin_user=$(ynh_app_setting_get --app=$app --key=admin)
 is_public=$(ynh_app_setting_get --app=$app --key=is_public)
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
-ynh_print_OFF
 upload_password=$(ynh_app_setting_get --app=$app --key=upload_password)
-ynh_print_ON
 phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
+datadir=$(ynh_app_setting_get --app=$app --key=datadir)
```

Comment:
`datadir` should be manage in the `ENSURE DOWNWARD COMPATIBILITY` step when it gets no value